src/components/ValidQuestion.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class='container'>
+    <div class='sideBar'>
+      <!--This is where the side bar goes-->
+    </div>
+
+    <div>
+      <keep-alive>
+        <router-view :articles="articles" :loaded="loaded"></router-view>
+      </keep-alive>
+    </div>
+  </div>
+</template>
+
+<script>
+import Sentiment from '../Sentiment.js'
+export default {
+
+  // theResponse will get the returned data
+  props: ['theResponse'],
+
+  data () {
+    return {
+      articles: null,
+      loaded: false
+    }
+  },
+
+  // Method for making the call to get articles from the cluster
+  // This will be called as soon as the component is ready
+  mounted: function () {
+    // First get the ticker from the response
+    var ticker = '/' + this.theResponse.companySymbol
+    var companyName = '/' + this.theResponse.companyName
+    var baseUrl = 'http://localhost:4040'
+
+    var fullUrl = baseUrl + ticker + companyName
+    this.$http.get(fullUrl).then(response => {
+      // Success
+      var resp = response.body
+      console.log(resp)
+      if (!resp.hasOwnProperty('solrErrorMessage')) {
+        var arrOfArticles = resp.response.docs
+        this.articles = Sentiment.sortBySentiment(arrOfArticles)
```

Comment:
As it stands right now unrelated articles are displaying on the page. After we've gotten the sentiment here we should definitely take into account the order in which the articles were returned from SOLR, as it dictates the articles relevance